.drone.star

```diff
@@ -8,6 +8,7 @@ dir = {
     "federated": "/var/www/owncloud/federated",
     "server": "/var/www/owncloud/server",
     "web": "/var/www/owncloud/web",
+    "owncloud-web": "owncloud/web",
```

Comment:
You should be able to use `ctx.repo.slug` here? Drone docs says: `Provides the user or organisation and the repository name`

Reply:
fixed